### DISCUSSION TITLE
TV端Netflix为什么看非自制局就说我用代理呢？我看之前几个都关闭也 也没有好的回答 我就来再问问

### DISCUSSION THREAD

**Discussion by doomzc** [vernesong/OpenClash#3963] [General]
我只是有个想法 是不是跟我的旁路由的关系？我两个家用n1 刷f大的固件都这样。两外一个弄了一个n100小主机pve就ok没遇到这个问题。都是我设置的 为啥差距这么大 我只是个小白 我好难

**Comment by vernesong** [comment]
节点不正常吧，插件流媒体有个自动检测
